chispa/schema_comparer.py

```diff
@@ -3,9 +3,11 @@
 import typing
 from itertools import zip_longest
 
+import six
```

Comment:
I don't think we need six; We ca use [itertools.zip_longest](https://github.com/MrPowers/chispa/blob/cccff0784017ab0b235951af5e21cfc196d1b0dc/chispa/rows_comparer.py#L3) from the standard library.

Reply:
updated!